all_in_one_seo_pack.php

```diff
@@ -511,3 +511,27 @@ function disable_all_in_one_free() {
 		}
 	}
 }
+
+spl_autoload_register( 'aioseop_autoloader' );
+/**
+ * Automatically loads classes for the plugin. Checks a namespace and loads only approved classes.
+ * Classes load from the /classes/ folder and need to be of the format 'aioseop_<name>.php'.
+ *
+ * @param string $class The class name to autoload.
+ *
+ * @return boolean Returns TRUE if the class is located. Otherwise FALSE.
+ */
+function aioseop_autoloader( $class ) {
+	$namespaces = array( 'aioseop' );
+	foreach ( $namespaces as $namespace ) {
+		if ( substr( $class, 0, strlen( $namespace ) ) == $namespace ) {
+			$filename = sprintf( '%s/classes/%s.php', dirname( __FILE__ ), $class );
```

Comment:
AIOSEOP_PLUGIN_DIR gives a trailing slash. sprintf looks more elegant with provided slashes otherwise it would have to be `%sclasses/%s.php` instead of `%s/classes/%s.php`

Reply:
It's usually a good idea to, but I can see your point.